doc/enterprise/read_views.rst

```diff
@@ -104,6 +104,39 @@ Similarly, you can retrieve data by the specific index.
       - [3, 'Ace of Base', 1987]
     ...
 
+Pagination is supported in read views in the same ways as in ``select`` requests
+to spaces: using the ``fetch_pos`` and ``after`` arguments. To get the cursor position
+after executing a request on a read view, set ``fetch_pos`` to ``true``:
+
+.. code-block:: tarantoolsession
+
+    tarantool> result, position = read_view1.space.bands:select({}, { limit = 3, fetch_pos = true })
+    ---
+    ...
+
+    tarantool> result
+    ---
+    - - [1, 'Roxette', 1986]
+      - [2, 'Scorpions', 1965]
+      - [3, 'Ace of Base', 1987]
+    ...
+
+    tarantool> position
+    ---
+    - kQM
+    ...
+
+Then, pass this position in the ``after`` parameter of a request to get the
+next data chunk:
+
+.. code-block:: tarantoolsession
+
+    tarantool> read_view1.space.bands:select({}, { limit = 3, after = position })
+    ---
+    - - [4, 'The Beatles', 1960]
+      - [5, 'Pink Floyd', 1965]
+      - [6, 'The Rolling Stones', 1962]
+    ...
 
 ADD example with fech_pos and after
```

Comment:
Seems irrelevant - the patch introduces such example.

Reply:
thanks, that's a bug. Fixed